rocAL/CMakeLists.txt

```diff
@@ -219,6 +224,9 @@ if(${BUILD_ROCAL})
     set(LINK_LIBRARY_LIST ${LINK_LIBRARY_LIST} ${FILESYSTEM_LIBRARIES})	
     # half
     include_directories(${HALF_INCLUDE_DIRS})
+    # SndFile
```

Comment:
Handle this similar to ffmpeg.

Reply:
@swetha097 Have we tested this in an environment were sndfile is not available, Please check if rocAL builds without any issue in that environment

Reply:
Tested with Non sndfile env, Added ROCAL_AUDIO flag for the code